### PR TITLE
Changes to Front Docs

### PR DESCRIPTION
**Pull request by SpecificKnot**
More edits to the README.md, CONTRIBUTING.md and removed TROUBLESHOOTING.md because it was moved to https://github.com/cmdr2/stable-diffusion-ui/wiki/Troubleshooting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ If Windows SmartScreen prevents you from running the program click `More info` a
 ## On Linux: 
 Run `./start.sh` (or `bash start.sh`) in a terminal.
 
-The installer will take care of whatever is needed. A friendly [Discord community](https://discord.com/invite/u9yhsFmEkB) will help you if you face any problems.
+The installer will take care of whatever is needed. If you face any problems, join the [Discord](https://discord.com/invite/u9yhsFmEkB) and ask fo assistance.
```
Comment (cmdr2) [inline README.md:33]: 1. `fo` -> `for`
2. I'd like to keep the "friendly" adjective before `Discord`, and the word "community", so `joint the friendly [Discord community] and ask for assistance..`.

File: README.md [hunk 2]
```diff
@@ -126,4 +128,10 @@ If you have any code contributions in mind, please feel free to say Hi to us on
 # Disclaimer
 The authors of this project are not responsible for any content generated using this interface.
 
-The license of this software forbids you from sharing any content that violates any laws, produce any harm to a person, disseminate any personal information that would be meant for harm, spread misinformation, or target vulnerable groups. For the full list of restrictions please read [the license](LICENSE). You agree to these terms by using this software.
+The license of this software forbids you from:
```
Comment (cmdr2) [inline README.md:131]: The license of this software forbids you from sharing any content that:
- violates any laws.
- produces any harm to a person
- ...

There's a subtle but important legal difference.

**Comment by cmdr2** [comment]
Thanks for this! I've added a few comments, looks good overall. :)

**Comment by SpecificKnot** [comment]
Thanks for the feedback. I'll make the adjustments. This is my first time contributing so feedback is great. 😄

File: README.md (hunk 1 above)
Comment (SpecificKnot) [reply README.md:33]: Changed to:

File: README.md [hunk 3]
```diff
@@ -40,51 +40,55 @@ The installer will take care of whatever is needed. A friendly [Discord communit
 
 # Easy for new users, powerful features for advanced users
 ## Features:
+
 ### User experience
-- **Hassle-free installation**: No Dependencies or Technical Knowledge Required. 1-click install for Windows 10/11 and Linux. *No dependencies*, no need for WSL or Docker or Conda or technical setup. Just download and run!
-- **Clutter-free UI**: a friendly and simple UI, while providing a lot of powerful features
+- **Hassle-free installation**: No Dependencies such as Docker or Conda or Technical Knowledge is required. Just download and run!
```
[the remark this answers is not in the document]
Comment (SpecificKnot) [reply README.md:45]: - **Hassle-free installation**: Does not require technical knowledge, does not require pre-installed software. Just download and run!

File: README.md (hunk 2 above)
Comment (SpecificKnot) [reply README.md:131]: Changed to: The license of this software forbids you from sharing any content that:
- Violates any laws.
- Produces any harm to a person or persons.
- Disseminates (spreads) any personal information that would be meant for harm.
- Spreads misinformation.
- Target vulnerable groups.

**Comment by cmdr2** [comment]
Looks great, thanks for your help! :)

**Pull request merged**